consensus/consensus.go

```diff
@@ -76,6 +61,16 @@ type Consensus struct {
 	prepareBitmap        *bls_cosi.Mask
 	commitBitmap         *bls_cosi.Mask
 
+	// Commits collected from view change
+	bhpSigs          map[common.Address]*bls.Sign // signature on m1 type message
```

Comment:
what are bhp and nil？

Reply:
bhp: blockHashPrepared  (the signature for m1 type message)
nil: the signature for m2 type message.
I am not very good at naming :(

Reply:
TODO: add description